src/components/nav/nav.tsx

```diff
@@ -0,0 +1,49 @@
+import { Image, Stack } from "@chakra-ui/react"
+import { Link, useResolvedPath, useMatch } from "react-router-dom"
+import { Center, HStack, Icon, Text } from "@chakra-ui/react"
+import { IconType } from "react-icons/lib"
+import { logoSvg } from "../.."
+
+interface NavItemProps {
+  path: string
+  label: string
+  icon: IconType
+}
+
+export function NavItem({ path, label, icon }: NavItemProps) {
```

Comment:
I combined the `nav.tsx` and `nav-item.tsx` components into this file, and put it in its own directory to match the other components.